tests/_support/Test/DataType/AbstractDataTypeTestCase.php

```diff
@@ -710,6 +710,18 @@ public function testObjects()
         $this->testDataHelper->assertObjects($this->testObject, 'objects', $this->seed);
     }
 
+    public function testReverseRelation()
+    {
+        // // this will create a couple of objects which can be used for references
+        $testObjects = TestHelper::createEmptyObjects();
+
+        $this->createTestObject('objects');
+
+        $this->refreshObject();
```

Comment:
```suggestion
Maybe so?
```suggestion
        $this->createTestObject('objects');
        $testObjects = $this->testObject->getObjects();

```
```

Reply:
Thanks for your inpu @blankse , I fixed the test.